CONTRIBUTING.md

```diff
@@ -52,33 +52,39 @@ yarn lint:fix
 yarn build
 ```
 
+If you want to test a specific package, provide it using the `filter` flag. For example:
+```
+turbo run test --filter=@meilisearch/instant-meilisearch
```

Comment:
You still have to be in the root dir, right?

Reply:
I improved the explaination: 


If you want to test a specific package, from the root directory run any of the above command with the `filter` flag.

Example:
```
turbo run test --filter=@meilisearch/instant-meilisearch
```

Reply:
perfect! 👯